src/pages/about/blog/linting-pit-of-success/index.mdx

```diff
@@ -0,0 +1,235 @@
+import {
+  BlogMarkdownLayout,
+  BlogImage,
+} from "../../../../components/BlogMarkdownLayout";
+import codeCycle from "./code-cycle.png";
+import workflow from "./workflow.png";
+import passTheSalt from "./pass-the-salt.png";
+import reviewFunnel from "./review-funnel.png";
+
+export const meta = {
+  title: "Linting into The Pit of Success",
+  date: "2025-09-19",
+  author: "Peter Stenger",
+  tags: ["Technical", "Development"],
+};
+
+Welcome to our first blog post! Over the next couple months, we are hoping to post feature deep-dives,
+technical challenges, and more -- stay tuned! If you want to be notified when a blog post is released, we will be posting these on our [LinkedIn](https://www.linkedin.com/company/prairielearn/).
+Follow us there to stay up to date!
+
+:warning: If you are not a developer, be warned -- this is highly technical stuff!
```

Comment:
The tone here feels off from me - it feels exclusionary, unwelcoming, and kind of confrontational?